src/Nethermind/Nethermind.DataMarketplace.Consumers/Deposits/Domain/DepositDetails.cs

```diff
@@ -72,6 +72,11 @@ public void SetConfirmationTimestamp(uint timestamp)
             ConfirmationTimestamp = timestamp;
         }
 
+        public void SetTransactionHash(Keccak transactionHash)
```

Comment:
why not setter here? looks strange, like it was something different and then back simple again